bundles/org.openhab.binding.dwdpollenflug/src/main/resources/ESH-INF/thing/region.xml

```diff
@@ -0,0 +1,162 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="dwdpollenflug"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<thing-type id="region">
+		<supported-bridge-type-refs>
+			<bridge-type-ref id="bridge" />
+		</supported-bridge-type-refs>
+		
+		<label>DWD Pollen count index - Region</label>
+		<description>Pollen count index for a region or partregion</description>
+
+		<channel-groups>
+			<channel-group id="alder" typeId="pollentype">
+				<label>Alder</label>
+				<description>Information for alder</description>
+			</channel-group>
+			<channel-group id="ambrosia" typeId="pollentype">
+				<label>Ambrosia</label>
+				<description>Information for ambrosia</description>
+			</channel-group>
+			<channel-group id="ash" typeId="pollentype">
+				<label>Ash tree</label>
+				<description>Information for ash tree</description>
+			</channel-group>
+			<channel-group id="birch" typeId="pollentype">
+				<label>Birch</label>
+				<description>Information for birch</description>
+			</channel-group>
+			<channel-group id="grasses" typeId="pollentype">
+				<label>Grasses</label>
+				<description>Information for grasses</description>
+			</channel-group>
+			<channel-group id="hazel" typeId="pollentype">
+				<label>Hazel</label>
+				<description>Information for hazel</description>
+			</channel-group>
+			<channel-group id="mugwort" typeId="pollentype">
+				<label>Mugwort</label>
+				<description>Information for mugwort</description>
+			</channel-group>
+			<channel-group id="rye" typeId="pollentype">
+				<label>Rye</label>
+				<description>Information for rye</description>
+			</channel-group>
+			<channel-group id="updates" typeId="updates"/>
+		</channel-groups>
+		
+		<properties>
+			<property name="region_id"/>
+			<property name="region_name"/>
+			<property name="partregion_name"/>
+		</properties>

```

Comment:
I don't think you should add these properties, they just reflect the configuration.

Reply:
I removed region_id! The region_name and partregion_name I want to leave there for control reasons for those who use text-configuration, whether the correct region is loaded.

Reply:
Do you agree leaving region_name and partregion_name as properties?